src/solver/variable/include/antares/solver/variable/economy/links/flowQuad.h

```diff
@@ -268,6 +269,8 @@ class FlowQuad: public Variable::IVariable<FlowQuad<NextT>, NextT, VCardFlowQuad
 private:
     uint pLinkGlobalIndex;
     uint pNbHours;
+    std::vector<std::vector<double>>* transitMoyenInterco;
```

Comment:
_Cosmetic_
About std::vector<std::vector<double>>* transitMoyenInterco;

Maybe renaming it into **transitMoyenInterco_** (trailing underscore) or **pTransitMoyenInterco** to tell is a private data member.